### ISSUE TITLE
SmartCardServices inconvenience on OSX

### ISSUE THREAD

**Issue by sammy007** [LedgerHQ/blue-app-monero#14]
Is it possible to somehow sneak into stock ifd-ccid.bundle and eliminate inconvenient process of adding vendor ID to it?

**Comment by johnalanwoods** [comment]
Agreed, system level hacks and disabling csrservice isn’t good for non tech audience.

**Comment by cslashm** [comment]
I work on a full removal of SmartCardServices usage.
